api/v1alpha1/clusterextension_types.go

```diff
@@ -77,15 +82,20 @@ type ClusterExtensionSpec struct {
 
 const (
 	// TODO(user): add more Types, here and into init()
-	TypeInstalled = "Installed"
-	TypeResolved  = "Resolved"
+	TypeInstalled      = "Installed"
+	TypeResolved       = "Resolved"
+	TypeHasValidBundle = "HasValidBundle"
+	TypeHealthy        = "Healthy"
```

Comment:
Introducing health checks becomes tricky when we have to deal with adding service account. And Health checks are not a part of v1 cut. So based on discussions, we should be good to remove `TypeHealthy` for now, and add it later when the time comes (as we have a clear architecture designed for SA inclusion in spec).

Reply:
Addressed by https://github.com/operator-framework/operator-controller/pull/854, which would merge into this branch soon!